NOTICE.md

```diff
@@ -0,0 +1,33 @@
+# Notices for Eclipse Collections
+
+This content is produced and maintained by the Project Leads and Committers of the Eclipse Collections project.
+
+* Project home: https://projects.eclipse.org/projects/technology.collections
+* Project Leads / Committers: https://projects.eclipse.org/projects/technology.collections/who
+
+## Trademarks
+
+Eclipse Collections is a trademark of the Eclipse Foundation. Eclipse, and the Eclipse Logo are registered trademarks of the Eclipse Foundation.
+
+## Copyright
+
+All content is the property of the respective authors or their employers. For more information regarding authorship of content, please consult the listed source code repository logs.
+
+## Declared Project Licenses
+
+This project is made available under the terms of the Eclipse Public License (EPL) v. 1.0 or Eclipse Distribution License (EDL) v. 1.0 which are available at the following locations.
+
+* EPL 1.0: https://www.eclipse.org/legal/epl-v10.html
+* EDL 1.0: https://www.eclipse.org/org/documents/edl-v10.html
+
+SPDX-License-Identifier: EPL-1.0 OR BSD-3 Clause
```

Comment:
a question for education: why say BSD-3 Clause instead of EDL-1.0?

Reply:
> a question for education: why say BSD-3 Clause instead of EDL-1.0?

I had the same question. I found the answer in the EDL 1.0 documents link.

https://www.eclipse.org/org/documents/edl-v10.php

Reply:
@waynebeaton Now that I look at it the SPDX License Identifier in the list from SPDX site, the identifier on the EDL 1.0 page is missing a dash. It should be `BSD-3-Clause`. If this is correct, then it should be fixed on the EDL 1.0 page as well.

Reply:
> a question for education: why say BSD-3 Clause instead of EDL-1.0?

It's a great question. I did ask the SPDX team to add the EDL to their list and their response was (correctly) that the EDL is really just the `BSD-3-Clause`, which is a template for a licence. Basically everybody who uses the `BSD-3-Clause` modifies it and having every single one of them have a separate SPDX code doesn't actually make anything easier.

SPDX has a means of specifying licences that aren't on their list. We could use `LicenseRef-EDL-1.0` as the SPDX code, but that would just add confusion as we'd have to explain what it means. We don't have this problem when we use `BSD-3-Clause`.

> the identifier on the EDL 1.0 page is missing a dash

Nice catch. There should never be spaces in the SPDX code. I'll fix that.